api/src/functions/migrateUsersVersion/index.ts

```diff
@@ -2,13 +2,14 @@ import { FnType } from "@functions/fnType";
 import { handlerPath } from "@libs/handlerResolver";
 
 export default (vpcConfig: FnType["vpc"]): FnType => {
+  const isProd = process.env.STAGE === "prod";
   return {
     handler: `${handlerPath(__dirname)}/app.default`,
     timeout: 30,
     events: [
       {
         schedule: {
-          rate: ["cron(0 6 ? * SUN *)"],
+          rate: [isProd ? "cron(*/5 * * * ? *)" : "cron(0 6 ? * SUN *)"],
```

Comment:
[dust] You may want to schedule it in prod for every 5 minutes from 6PM to 6AM, something like:

```typescript
"cron(*/5 18-23,0-5 * * ? *)"
```

Additionally, we have more than 25 users in our other environments, so scheduling the lambda to run only once on Sundays may not be good. How about every 5 minutes from midnight to 6AM on Sunday?

```typescript
"cron(*/5 0-5 ? * SUN *)"
```